### PR TITLE
Update the guide to use JakartaEE 9 with javax-annotations-api

### PR DESCRIPTION
**Pull request by gkwan-ibm**
address issue #21 
Update the guide to use JakartaEE 9 with ~~tomcat-annotations-api~~ javax-annotations-api

### CODE REVIEW

File: finish/systemproto/pom.xml [hunk 1]
```diff
@@ -36,6 +37,13 @@
             <scope>provided</scope>
         </dependency>
         <!-- end::grpc-stub[] -->
+        <!-- grpc compile dependencies for Jakarta EE 9+ -->
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
```
Comment (hutchig) [inline finish/systemproto/pom.xml]: I don't think we should ship this without Alasdair's approval.

**Review by hutchig — CHANGES_REQUESTED** [review]
Need to check use of tomcat: See https://github.com/OpenLiberty/open-liberty/issues/23922

File: finish/systemproto/pom.xml (hunk 1 above)
Comment (hutchig) [reply finish/systemproto/pom.xml]: Perhaps liberty should bundle this annotation itself? See https://github.com/OpenLiberty/open-liberty/issues/23922

Comment (gkwan-ibm) [reply finish/systemproto/pom.xml]: Updated to use
```
        <dependency>
            <groupId>javax.annotation</groupId>
            <artifactId>javax.annotation-api</artifactId>
            <version>1.3.2</version>
```

**Review by ReeceNana — APPROVED** [review]

**Review by ReeceNana — APPROVED** [review]

File: finish/systemproto/pom.xml [hunk 2]
```diff
@@ -37,13 +37,14 @@
             <scope>provided</scope>
         </dependency>
         <!-- end::grpc-stub[] -->
-        <!-- grpc compile dependencies for Jakarta EE 9+ -->
+        <!-- tag::javax[] -->
         <dependency>
-            <groupId>org.apache.tomcat</groupId>
-            <artifactId>tomcat-annotations-api</artifactId>
-            <version>9.0.70</version>
+            <groupId>javax.annotation</groupId>
```
Comment (hutchig) [inline finish/systemproto/pom.xml:42]: I think this is much better from a app platform point of view

**Review by hutchig — APPROVED** [review]

**Comment by hutchig** [comment]
One more, trivial, comment.
I noticed that your more recent commits are not signed commits.
In development now, it is best practice for all commits going forward to be signed.

Also, although personally as an author I prefer daily etc commits for off-laptop check-in safety, I think we are encouraged to 'squash' commits into a few meaningful 'chunks' for review (and 1 per post review issue/reviewer) rather than retain the whole list of the operational-chunked commits in the gitlog of the main branch. Not important but worth becoming comfortable with squashing.

**Pull request merged**